public/app/features/admin/Users/OrgUsersTable.tsx

```diff
@@ -95,7 +94,7 @@ export const OrgUsersTable = ({
       {
         id: 'avatarUrl',
         header: '',
-        cell: ({ cell: { value } }: Cell<'avatarUrl'>) => <Avatar src={value} alt="User avatar" />,
+        cell: ({ cell: { value } }: Cell<'avatarUrl'>) => <Avatar src={value!} alt="User avatar" />,
```

Comment:
Instead of lying to typescript, just guard this properly `value && <Avatar src={value} alt="User avatar" />` ❤️